### ISSUE TITLE
Playback position of file isn't saved when using playlist.

### ISSUE THREAD

**Issue by negativeions** [lhc70000/iina#1034]
- macOS 10.12.6
- IINA 0.0.12 (build 37)

Not sure if this was mentioned already but playback position is only saved for files when they're opened manually. If you cycle hotkey or click on them in the playlist their last position is not saved. Thanks.

**Comment by lhc70000** [comment]
Yes, it's an intended behavior and is consistent with mpv. It only saves playback position when you **quit the player or closed the window**. If you switch to another video manually, the player thinks you are leaving the current video on purpose, thus it clears cache file for the video.

I'm not sure whether IINA can override this behavior as it needs more investigation. I'll keep an eye on this issue.
